use-package.el

```diff
@@ -1406,6 +1408,67 @@ deferred until the prefix key sequence is pressed."
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;
+;;; :customize
+;;
+
+(defun use-package-normalize/:customize (name-symbol keyword arg)
+  "Normalize use-package customize keyword."
+  (let ((error-msg (format  "%s wants a (<symbol> <form> <optional string comment>) or list of these" name-symbol)))
+    (unless (listp arg)
+      (use-package-error error-msg))
+    (dolist (def arg arg)
+      (unless (listp def)
+        (use-package-error error-msg))
+      (seq-let [variable value comment] def
```

Comment:
Emacs 24 and older do not include `seq.el`.

Reply:
Are there some other form of destructuring binding I could use ?

Reply:
you could  try `cl-destructuring-bind`?